public/app/plugins/datasource/mysql/sqlCompletionProvider.ts

```diff
@@ -1,267 +1,32 @@
-    resolve: (token, previousKeyword) => {
-      const is =
-        isDirectlyAfter(token, Keyword.Select) ||
-        (isAfterSelect(token) && token?.previous?.is(TokenType.Delimiter, ','));
-      return Boolean(is);
-    },
-  },
-];
-
-export const customSuggestionKinds: (
-  getTables: CompletionProviderGetterArgs['getTables'],
-  getFields: CompletionProviderGetterArgs['getColumns'],
-  fetchMeta: CompletionProviderGetterArgs['fetchMeta']
-) => SuggestionKindProvider = (getTables, _, fetchMeta) => () =>
-  [
-    {
-      id: CustomSuggestionKind.TablesWithinDataset,
-      applyTo: [CustomStatementPlacement.AfterDataset],
-      suggestionsResolver: async (ctx) => {
-        const tablePath = ctx.currentToken ? getTablePath(ctx.currentToken) : '';
-        const t = await getTables.current(tablePath);
-        return t.map((table) => suggestion(table.name, table.completion ?? table.name, CompletionItemKind.Field, ctx));
-      },
-    },
-    {
-      id: 'metaAfterSelect',
-      applyTo: [CustomStatementPlacement.AfterSelect],
-      suggestionsResolver: async (ctx) => {
-        const path = getPath(ctx.currentToken, Direction.Next);
-        const t = await fetchMeta.current(path);
-        return t.map((meta) => {
-          const completion = meta.kind === CompletionItemKind.Class ? `${meta.completion}.` : meta.completion;
-          return suggestion(meta.name, completion!, meta.kind, ctx);
-        });
-      },
-    },
-    {
-      id: 'metaAfterSelectFuncArg',
-      applyTo: [StatementPosition.AfterSelectFuncFirstArgument],
-      suggestionsResolver: async (ctx) => {
-        const path = getPath(ctx.currentToken, Direction.Next);
-        const t = await fetchMeta.current(path);
-        return t.map((meta) => {
-          const completion = meta.kind === CompletionItemKind.Class ? `${meta.completion}.` : meta.completion;
-          return suggestion(meta.name, completion!, meta.kind, ctx);
-        });
-      },
-    },
-    {
-      id: 'metaAfterFrom',
-      applyTo: [CustomStatementPlacement.AfterFrom],
-      suggestionsResolver: async (ctx) => {
-        // TODO: why is this triggering when isAfterFrom is false
-        if (!isAfterFrom(ctx.currentToken)) {
-          return [];
-        }
-        const path = ctx.currentToken?.value || '';
-        const t = await fetchMeta.current(path);
-        return t.map((meta) => suggestion(meta.name, meta.completion!, meta.kind, ctx));
-      },
-    },
-    {
-      id: `MYSQL${StatementPosition.WhereKeyword}`,
-      applyTo: [StatementPosition.WhereKeyword],
-      suggestionsResolver: async (ctx) => {
-        const path = getPath(ctx.currentToken, Direction.Previous);
-        const t = await fetchMeta.current(path);
-        return t.map((meta) => {
-          const completion = meta.kind === CompletionItemKind.Class ? `${meta.completion}.` : meta.completion;
-          return suggestion(meta.name, completion!, meta.kind, ctx);
-        });
```

Comment:
Did this custom handling work? If so, is the support dropped now?

Reply:
it did but I think it did the same as what we have in place. showing the columns after WHERE.